backend/src/case_file/entities/case_file.entity.ts

```diff
@@ -2,8 +2,15 @@ import { Assessment } from "./assessment.entity";
 import { Prevention } from "./prevention.entity";
 
 export class CaseFile {
-    caseIdentifier: string;
+    caseIdentifier?: string;
     leadIdentifier: string;
-    assessmentDetails: Assessment;
-    preventionDetails: Prevention;
+    assessmentDetails?: Assessment;
+    preventionDetails?: Prevention;
+    isReviewRequired?: boolean;
+    reviewComplete?: {
```

Comment:
instead of an inline type, I would recommend creating a new type the way that there is for the assessment and prevention